### ISSUE TITLE
Disabled buttons

### ISSUE THREAD

**Issue by KicKerBNU** [beercss/beercss#28]
Should we work on a disabled css state for buttons on BeerCss ? Today they look active when disabled

Found this example on Material Design Docs
![image](https://user-images.githubusercontent.com/43939417/148402311-17e31942-bef6-45a9-86bd-80491997a5ca.png)

**Comment by leonardorafael** [comment]
Really nice @KicKerBNU, this needs to be in the next release. 💯

**Comment by laurentpayot** [comment]
+1. I had to use the following style for my buttons: 
```css
button:disabled {
    opacity: 0.5;
    cursor: not-allowed;
}
```
Do you agree with the `cursor: not-allowed` property?

**Comment by leonardorafael** [comment]
Released in v2.0.0 🍺🎉
